packages/golden_toolkit/test/configuration_test.dart

```diff
@@ -173,6 +175,40 @@ void main() {
       );
     });
 
+    group('Shadows', () {
+      GoldenToolkit.runWithConfiguration(
+        () {
+          testGoldens(
+              'screenMatchesGolden method uses enableRealShadows from global configuration when GoldenToolkit.runWithConfiguration is outside testGoldens',
+              (tester) async {
+            await tester.pumpWidgetBuilder(WidgetWithShadows());
+            await screenMatchesGolden(
+                tester, 'enableRealShadows_honored_when_testGoldens_wrapped');
```

Comment:
you'll see that `enableRealShadows_honored_when_testGoldens_wrapped.png` has nice shadows :)